src/server/worldserver/worldserver.conf.dist

```diff
@@ -1590,15 +1590,23 @@ IsPreloadedContinentTransport.Enabled = 0
 #        Default:     0 - (Disabled)
 #                     1 - (Enabled)
 
-Warden.Enabled = 1
+Warden.Enabled = 0
```

Comment:
Change to **Warden.Enabled = 1**

Reply:
NO! i synchonized config setting with which is defaulted in core.

Reply:
Also 
#        Default:     0 - (Disabled)
so it is correct, and having it enabled by default is not neccessary